.gitignore

```diff
@@ -22,3 +22,4 @@ build/
 _yardoc
 doc/
 .idea/
+vendor
```

Comment:
when running `bundle exec middleman server` locally to view the files, the tool creates this folder, so I added it to the gitignore